### DISCUSSION TITLE
Markdown preview improvements

### DISCUSSION THREAD

**Discussion by sebastianjacobs** [standardnotes/forum#2778] [Suggestions]
- No syntax highlighting: It would help me a lot if I could add ` ```php ` to get PHP syntax highlighting for example.
- A `inline code` element that goes over one line does overlap and makes it harder to read.
- More spacing on top and bottom of a ` --- ` element: Not so important but it would look much better with more spacing.

**Comment by stefan-yas** [comment]
Thanks for the suggestions, @sebastianjacobs - what Markdown editor are you using?

**Comment by sebastianjacobs** [comment]
I use Markdown Pro, but it's the same with Markdown Basic.

**Comment by jrcacd** [comment]
It would be very useful to have in markdown Alternative the highlight `===Text===`

**Comment by sebastianjacobs** [comment]
> * No syntax highlighting: It would help me a lot if I could add `` ```php `` to get PHP syntax highlighting for example.
> * A `inline code` element that goes over one line does overlap and makes it harder to read.
> * More spacing on top and bottom of a `---` element: Not so important but it would look much better with more spacing.

@stefan-yas Is this still on your radar and planned in the near future?

**Comment by stefan-yas** [comment]
> > * No syntax highlighting: It would help me a lot if I could add `` ```php `` to get PHP syntax highlighting for example.
> > * A `inline code` element that goes over one line does overlap and makes it harder to read.
> > * More spacing on top and bottom of a `---` element: Not so important but it would look much better with more spacing.
> 
> @stefan-yas Is this still on your radar and planned in the near future?

Hi there, I'm no longer affiliated with the Standard Notes organization, but I'm sure someone from the team will get back to you on that. Happy note-taking!
